create_tarball.sh

```diff
@@ -49,12 +50,26 @@ if [ -d ${pilot_version}/software/${os}/${cpu_arch_subdir}/modules ]; then
     find ${pilot_version}/software/${os}/${cpu_arch_subdir}/modules -type f | grep -v '/\.wh\.' >> ${files_list}
     # module symlinks
     find ${pilot_version}/software/${os}/${cpu_arch_subdir}/modules -type l | grep -v '/\.wh\.' >> ${files_list}
+    # module files and symlinks
+    find ${pilot_version}/software/${os}/${cpu_arch_subdir}/modules/all -type f -o -type l \
+        | grep -v '/\.wh\.' | sed -e 's/.lua$//' | awk -F'/' '{printf "%s/%s\n", $(NF-1), $NF}' | sort | uniq \
```

Comment:
avoid hardcoding standard EasyBuild module naming scheme by using `sed 's@.*/modules/all/@@g'`

use `sort -u` instead of `sort | uniq`

Reply:
Changed via https://github.com/EESSI/software-layer/pull/239/commits/4d132273569f6d40e503bdd48b5b3aca88f7c907